docs/installation.rst

```diff
@@ -43,38 +42,47 @@ path, and that you can verify by doing:
 
    $ which raven
 
-For development purposes, you can add
-``--install-option="--with-testdata"`` to the above ``pip install``
-command, which will also download a snapshot of the the `Raven Test
-Data <https://github.com/Ouranosinc/raven-testdata>`_ into your venv.
+If you don't use the ``--install-option="--with-raven"`` option or if
+there's a problem with it, you can also manage the necessary binaries
+yourself (see the next section).
 
 
-From sources
-------------
+From sources (for development and testing)
+------------------------------------------
 
 The sources for RavenPy can be obtained from the GitHub repo:
 
 .. code-block:: console
 
     $ git clone git://github.com/CSHS-CWRA/ravenpy
 
-You can then follow the instructions above to setup your venv, but install with:
+You must download and install `Raven <http://raven.uwaterloo.ca>`_ and
+`OSTRICH <http://www.civil.uwaterloo.ca/envmodelling/Ostrich.html>`_
+and they must be available on your PATH. Alternatively, you can supply
+``RAVENPY_RAVEN_BINARY_PATH`` and ``RAVENPY_OSTRICH_BINARY_PATH`` env
+variables to your Python interpreter.
+
+Once you have created and activated your venv, you can do:
 
 .. code-block:: console
 
    $ cd /path/to/ravenpy
-   $ pip install . --verbose --install-option="--with-raven"
+   $ pip install --verbose --editable .
 
-If you want to run the tests, install with:
+If you want to run the tests, install the requirements:
 
 .. code-block:: console
 
-   $ cd /path/to/ravenpy
-   $ pip install . --verbose --install-option="--with-raven" --install-option="--with-testdata"
    $ pip install -r requirements_dev
```

Comment:
```suggestion
   $ pip install --editable .[dev]
   or 
   $ pip install -r requirements_dev
```

Reply:
Didn't know I could do it like that, I like it!

Reply:
If users have a non-standard shell (`zsh`, `fish`), it needs to be `".[dev]"`, but if they're using a non-standard shell, they probably know this already.